.golangci.yml

```diff
@@ -13,6 +13,7 @@ linters:
     - funlen
     - wsl
     - gomnd
+    - maligned
```

Comment:
is this part of this PR?

Reply:
yeah, it's a check that takes a lot of time and it'd fail because I'm modifying structs.